### PR TITLE
Accessibility Updates

### LINKED ISSUE #78: Accessibility Updates
We'd like to make the following accessibility updates to this component
- Allow customization of the numbered page button titles through the locale
- have screen readers interpret the `Next Page` and `Previous Page` as disabled when they are disabled.
Would these updates be ok?

### PR DESCRIPTION
**Pull request by geramirez**
#61 would not work for this. We believe commit cd4fa10 will. 
(Resolves #78) 

---

Close: https://github.com/react-component/pagination/issues/78

### CODE REVIEW

File: src/Pagination.jsx [hunk 1]
```diff
@@ -212,6 +212,7 @@ class Pagination extends React.Component {
             title={props.showTitle ? locale.prev_page : null}
             onClick={this._prev}
             className={`${this._hasPrev() ? '' : `${prefixCls}-disabled`} ${prefixCls}-prev`}
+            aria-disabled={!this._hasPrev()}
```
Comment (benjycui) [inline src/Pagination.jsx]: +1 for this.

File: src/Pager.jsx [hunk 2]
```diff
@@ -13,8 +14,12 @@ const Pager = (props) => {
     cls = `${cls} ${props.className}`;
   }
 
+  if (props.locale.page_prefix) {
```
Comment (benjycui) [inline src/Pager.jsx]: But I don't think this is a good idea.

In Chinese: `第 1 页`

In English: `Page 1`

So, `props.locale.page_prefix` cannot cover those use case.

Just use https://github.com/react-component/pagination/issues/61

So:

```jsx
<Pagination itemRender={(page) => <li title={`第 ${page} 页`} />} />
```

Comment (geramirez) [inline src/Pager.jsx]: Ok, no problem. Let me remove it

**Comment by geramirez** [comment]
ok, I've made the updates and will use #61

**Comment by benjycui** [comment]
```bash
➜  ~ npm owner ls rc-pagination
afc163 <afc163@gmail.com>
ibigbug <xiaobayuwei@gmail.com>
paranoidjk <hust2012jiangkai@gmail.com>
yiminghe <yiminghe@gmail.com>
```

File: src/Pagination.jsx [hunk 3]
```diff
@@ -233,6 +234,7 @@ class Pagination extends React.Component {
             title={props.showTitle ? locale.next_page : null}
             onClick={this._next}
             className={`${this._hasNext() ? '' : `${prefixCls}-disabled`} ${prefixCls}-next`}
+            aria-disabled={!this._hasNext()}
```
Comment (paranoidjk) [inline src/Pagination.jsx]: i will do a little change，to make sure _hasNext will only be called 1 time during every render

**Comment by paranoidjk** [comment]
@geramirez 1.8.8

**Comment by geramirez** [comment]
Thanks for the change

File: src/Pager.jsx (hunk 2 above)
Comment (antoniojps) [reply src/Pager.jsx]: Mhm I know this super old, but how I setup the itemrRender to be exactly the default one, but with the custom title?

**Pull request merged**